### PR TITLE
Fix GPU Temperature Sensor

### LINKED ISSUE #203: [bug] GPU Temperature not showing
GPU temperature is not showing.

image

 - eul version: 1.5.18
 - macOS version: 11.4
 - Device model: Macbook pro 2018

### PR DESCRIPTION
**Pull request by huijiewei**
fixed #203

### CODE REVIEW

File: eul/Utilities/GPU.swift [hunk 1]
```diff
@@ -63,12 +63,18 @@ extension GPU {
...
+                temperature = SmcControl.shared.gpuProximityTemperature
+            }
+
             Print("📊 statistics", statistics)
 
             return Statistic(
                 pciMatch: pciMatch,
                 usagePercentage: usagePercentage,
-                temperature: statistics["Temperature(C)"] as? Double ?? SmcControl.shared.gpuProximityTemperature,
+                temperature: temperature,
```
Comment (gao-sun) [inline eul/Utilities/GPU.swift:77]: can we use `?:` here to avoid using `var`? i would prefer `let`.

**Review by gao-sun — COMMENTED** [review]
thanks, would you mind to remove version bumping? i'll do it separately.

File: eul/Utilities/GPU.swift (hunk 1 above)
Comment (huijiewei) [reply eul/Utilities/GPU.swift:77]: i am swift beginner，you should patch it self，thanks

**Comment by huijiewei** [comment]
sorry i am beginner，you can close pr and patch it self

sorry for my english

**Pull request closed without merging**